### DISCUSSION TITLE
Issue connecting with metamask

### DISCUSSION THREAD

**Discussion by Jrosado16** [rainbow-me/rainbowkit#1533] [Feedback]
I am trying to connect with MM but I never get the MM option to connect and when I click on the icon it takes a while to go to the wallet

https://github.com/rainbow-me/rainbowkit/assets/32801851/e54a7ccb-4a70-43e7-9c3f-14a65e26250c

**Comment by 0xmkh** [comment]
@Jrosado16 How does your Wagmi and RainbowKit setup look like ?
